component/modal/modal.css

```diff
@@ -0,0 +1,37 @@
+.modal-container{
+  
+    max-width: 600px;
+    width: 100%;
+    padding:1rem;
+    overflow: auto;
+    padding: 0px 30px;
+    box-sizing: border-box;
+    background-color:var(--dark-blue);
+    color:white;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+}
+.modal-content-ctn{
+    text-align: center;
+}
```

Comment:
Can you please explain what does this class do? Cannot understand what **-ctn** mean

Reply:
by default it is getting left aligned and I want to align it at center!! 
And ctn is used for container.

Reply:
using modal-center is more meaningful than using .modal-ctn

Reply:
I have changed it to modal-container is this would be fine ??

Reply:
Yeah modal-container sounds more meaningful 😊

Reply:
Can you please approve it so that I could merge this branch ?? Or do I have to make more changes ??

Reply:
yeah doing it😊